<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChambreUtilisateur</class>
 <widget class="QDialog" name="ChambreUtilisateur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> Hotel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/projetimages/projetimages/e.png</normaloff>:/projetimages/projetimages/e.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableView {
selection-background-color:red;
border-radius:15px;
}

QDialog
{
	background-image:url(:/projetimages/projetimages/a.jpg);

}
*{
font-family:centry gothlic ;
font-size:15px;
}
#frame
{

	background:rgba(0,0,0,0.7);
border-radius:15px;
}
 
QLabel
{
color:white;
background:transparent;

}


QLineEdit
{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>921</width>
     <height>451</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTableView" name="tableViewchutil">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>301</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Saisir numéro de chambre à chercher :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="chambrechercher_txt">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Chambres disponibles :</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewchdispoutil">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>291</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
